### ISSUE TITLE
perl6 htmlify.p6 --no-proc-async - dies

### ISSUE THREAD

**Issue by coke** [perl6/doc#1152]
```
...
 243/244: doc/Type/X/TypeCheck/Return.pod6         => type/X::TypeCheck::Return
 244/244: doc/Type/X/TypeCheck/Splice.pod6         => type/X::TypeCheck::Splice
No such method 'started' for invocant of type 'Any'
  in method throw at /Users/williamcoleda/.rakudobrew/moar-e182deb529/install/share/perl6/runtime/CORE.setting.moarvm line 1
  in sub highlight-code-blocks at htmlify.p6 line 967
  in sub MAIN at htmlify.p6 line 183
  in block <unit> at htmlify.p6 line 1005
```

on OS X 10.12.2, rakudo 2017.01

**Comment by coke** [comment]
Changing line 967 to read

```
if $proc.DEFINITE && !$proc.started {
```
seems to help.

**Comment by coke** [comment]
fixed in 4a8f8f5af5f25ef44d4c4472a5a319e01f807b96
